app/src/main/kotlin/com/wire/android/ui/calling/CallActivity.kt

```diff
@@ -20,14 +20,14 @@ package com.wire.android.ui.calling
 import android.content.Intent
 import android.os.Build
 import android.view.WindowManager
+import androidx.activity.ComponentActivity
 import androidx.activity.viewModels
-import androidx.appcompat.app.AppCompatActivity
 import androidx.lifecycle.lifecycleScope
 import com.wire.android.ui.AppLockActivity
 import com.wire.kalium.logic.data.id.QualifiedIdMapperImpl
 import kotlinx.coroutines.launch
 
-abstract class CallActivity : AppCompatActivity() {
+abstract class CallActivity : ComponentActivity() {
```

Comment:
What was the reason to switch `CallActivity` from `AppCompatActivity` to `ComponentActivity`?

Reply:
It's a leftover I was trying using `onUserLeaveHint`

Reply:
`onUserLeaveHint` is not supported in `AppCompatActivity`